src/modules/common/components/admin-toggle.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const AdminToggle = ({ adminMode, toggleAdminMode }) => {
+  const toggleClass = adminMode ? 'footer-admin-toggle--active' : '';
+
+  return (
+    <label
+      className={`footer-admin-toggle ${toggleClass}`}
+      htmlFor="adminMode"
+    >
+      <input type="checkbox" checked={adminMode} value={adminMode} onClick={toggleAdminMode} />{' '}
```

Comment:
If I check the footer box, then refresh, in console I get:
```
Warning: Failed form propType: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`. Check the render method of `AdminToggle`.
```
If I change the `onClick` to `onChange` the warning goes away, but I'm not sure there's a negative consequence?